experiment/secret-sync-controller/secret-sync-config.go

```diff
@@ -0,0 +1,72 @@
+package main
```

Comment:
I would expect this to live in something other than main, I think that might get you out of this error

```
$ go test ./experiment/secret-sync-controller/main_test.go
# command-line-arguments [command-line-arguments.test]
experiment/secret-sync-controller/main_test.go:84:13: undefined: SecretSyncSpec
```

Reply:
Resolved.
Put `SecretSyncConfig` into the `config` package, and  `SecretSyncController` into the `controller` package.